beacon-chain/state/state-native/getters_state.go

```diff
@@ -338,3 +400,11 @@ func ProtobufBeaconStateCapella(s interface{}) (*ethpb.BeaconStateCapella, error
 	}
 	return pbState, nil
 }
+
+func ProtobufBeaconStateDeneb(s interface{}) (*ethpb.BeaconStateDeneb, error) {
+	pbState, ok := s.(*ethpb.BeaconStateDeneb)
+	if !ok {
+		return nil, errors.New("input is not type pb.BeaconStateCapella")
```

Comment:
the error here should be StateDeneb I think

Reply:
a702121fb159098ee60700edaa70b8db08065699